src/itest/java/uk/gov/companieshouse/company/metrics/steps/TestSupport.java

```diff
@@ -22,7 +22,7 @@ public class TestSupport {
     public static final String RESOURCE_KIND = "company-charges";
     public static final String CONTEXT_ID = "context_id";
     public static final String RESOURCE_ID = "11223344";
-    public static final String TYPE = "charges";
+    public static final String TYPE = "changed";
```

Comment:
Would this be best set to `CHANGED_TYPE` rather than `TYPE`? And `DELETED_TYPE` instead of `DELETE_TYPE`?

Reply:
I agree that's much better.